library-best-practices/JLBP-3.md

```diff
@@ -36,7 +36,7 @@ Definition of semantic versioning (SemVer): https://semver.org
     features) that is exposed through a library's public API
   - Adding a new class
   - Adding a new method
-- Special case: Merely bumping up the minimum required Java version (and not
-  making any breaking surface changes) does not necessarily mean that a
-  library should bump its major version, because new Java versions break
-  very little surface from prior versions.
+- Special case: Maintainers need not increment a library's major version when
```

Comment:
need not --> do not have to

This sentence is long. Can it broken up?

Reply:
Removed "a new release does not make any breaking surface change" in favor of "a release is only to drop..."